bin/BuildPackages.sh

```diff
@@ -225,6 +229,26 @@ build_one_package() {
       mkdir -p bin && test -x bin/bistellar || mv bistellar bin
     ;;
 
+    digraphs*)
+      CONFIGFLAGS_OLD="$CONFIGFLAGS"
+      if [[ $WITH_EXTERNAL_PLANARITY = yes ]]
+      then
+        CONFIGFLAGS="--with-external-planarity $CONFIGFLAGS"
+      fi
+      run_configure_and_make
+      CONFIGFLAGS="$CONFIGFLAGS_OLD"
```

Comment:
I didn't test it, but I suspect this change breaks the error detection; i.e. with it, an error during the build of digraph and semigroups won't be detected, due to its multiline nature. I am afraid this is a deal breaker. But otherwise, this PR seems fine to me.